### PR TITLE
Improve the documentation about how to delete over-resource-used pods…

### PR DESCRIPTION
**Pull request by jiasheng55**
… in elastic quota management (#138)

### CODE REVIEW

File: docs/user-manuals/multi-hierarchy-elastic-quota-management.md [hunk 1]
```diff
@@ -616,6 +618,12 @@ rules:
   - patch
   - update
   - delete
```
Comment (jiasheng55) [inline docs/user-manuals/multi-hierarchy-elastic-quota-management.md:620]: Not sure if the "delete" verb of pods should be removed or not, help me here please.

File: docs/user-manuals/multi-hierarchy-elastic-quota-management.md [hunk 2]
```diff
@@ -616,6 +618,12 @@ rules:
...
+- apiGroups:
+  - ""
+  resources:
+  - pods/eviction
```
Comment (jiasheng55) [inline docs/user-manuals/multi-hierarchy-elastic-quota-management.md:624]: Actually I'm not sure if this should be installed by default. Why do we need users to configure this manually:)

**Comment by hormes** [comment]
@buptcozy please review this pr

**Review by buptcozy — APPROVED** [review]
/approve

**Comment by jiasheng55** [comment]
@hormes @buptcozy @eahydra Could you help to finish the review process.

**Comment by eahydra** [comment]
@jiasheng55 hi, missing DCO, you can append signature as follows
```sh
$ git commit -s --amend --no-edit
$ git push -f
```

File: docs/user-manuals/multi-hierarchy-elastic-quota-management.md (hunk 2 above)
Comment (eahydra) [reply docs/user-manuals/multi-hierarchy-elastic-quota-management.md:624]: We will add the RBAC in the latest version.

File: docs/user-manuals/multi-hierarchy-elastic-quota-management.md (hunk 1 above)
Comment (eahydra) [reply docs/user-manuals/multi-hierarchy-elastic-quota-management.md:620]: `Delete` is used by default preemption.

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**